src/clientprocess/clientprocess_main.jl

```diff
@@ -34,15 +34,12 @@ while true
             end
             serialize(stdout, (:success, pkgs))
         elseif message == :load_package
-            ostdout = stdout
-            (outRead, outWrite) = redirect_stdout() # seems necessary incase packages print on startup
-            SymbolServer.import_package(payload, depot)
+            redirect_stdout(open("/dev/null", "w")) do # seems necessary incase packages print on startup
```

Comment:
`/dev/null` doesn't work on Windows...

Maybe just sticking with the old code but adding some `try finally` would also do the trick?

Reply:
Oh yeah, forgot about that... I see a few options:
 1. We could check the platform and use `"nul"` special file on Windows
 2. We could add inner try/catch/finally block in the `:load_package` case, rethrow the error in `catch` and redirect `stdout` in `finally`
 3. We could simply redirect `stdout` to `ostdout` in the catch block regardless if it was redirected earlier or not (don't know if not closing the pipe ends would be a problem)

Reply:
I like the `nul` option, that seems cleanest!